app/com/gu/friendly/tailor/EventProcessor.scala

```diff
@@ -59,26 +67,28 @@ object EventProcessor extends LazyLogging {
   def deserializeToEvent(record: Record): Event =
     ThriftSerializer.fromByteBuffer(record.getData)(Event.decoder)
 
-  def processPageView(ev: Event) = for {
+  def relevantPageView(ev: Event):Option[RelevantPageView] = for {
```

Comment:
This could be an `apply()` method on the `RelevantPageView` companion object

https://twitter.github.io/scala_school/basics2.html#apply